src/coreclr/src/pal/src/libunwind/src/CMakeLists.txt

```
@@ -51,6 +51,10 @@ elseif(CLR_CMAKE_HOST_ARCH_ARM64)
         # Disable warning due to labs function called on unsigned argument
         add_compile_options(-Wno-absolute-value)
     endif()
+    // Issue https://github.com/libunwind/libunwind/issues/176
+    if (CLR_CMAKE_TARGET_ALPINE_LINUX)
```

it only applies to arm64:

```suggestion
    if (CLR_CMAKE_TARGET_ALPINE_LINUX AND CLR_CMAKE_TARGET_ARCH_ARM64)
```

---

It already in a ARM64 block.